### ISSUE TITLE
train a sentiment analysis classifier with a non-nn model

### ISSUE THREAD

**Issue by LucaNyckees** [LucaNyckees/recommendation_system#7]
### Problem

We wish to create a sentiment analysis classifier (positive, neutral, negative) by exploring non-neural network models, such as a gradient boosting model or a random forest. As input, data would have been embedded via an pre-trained embedding.

### ToDo

- [ ] implement framework to choose a model and to compare results, where the task is reviews sentiment learning

### Context

Exploration.
